packages/editor-ui/src/components/WorkflowSettings.vue

```diff
@@ -23,7 +23,7 @@
 							placeholder="Select Execution Order"
 							size="medium"
 							filterable
-							:disabled="readOnlyEnv"
+							:disabled="readOnlyEnv || !workflowPermissions.update"
```

Comment:
This condition seems to be repeated quite a lot. Could you extract it into a computed prop?